code/modules/admin/verbs/datumvars.dm

```diff
@@ -693,6 +693,8 @@
 			return
 
 		M.fully_replace_character_name(M.real_name, new_name)
+		vv_update_display(M, "name", new_name)
```

Comment:
shouldnt this be in #1384

Reply:
This is VV/admin tools related, that one is for a game proc